inst/templates/ad_adlbhy.R

```diff
@@ -43,7 +43,7 @@ adlb_annotated <- adlb %>%
       )
     )
   ) %>%
-  select(STUDYID, USUBJID, TRT01A, PARAMCD, LBSEQ, ADT, AVISIT, ADY, AVAL, ANRHI, CRIT1, CRIT1FL)
+  select(USUBJID, STUDYID, TRT01A, PARAMCD, LBSEQ, ADT, AVISIT, ADY, AVAL, ANRHI, CRIT1, CRIT1FL)
```

Comment:
why was this one left behind with just the order of USUBJID and STUDYID swapped? noticed this by chance as was using this PR to prompt same changes in admiralonco. please double check any others

Reply:
as FYI should be `select(!!!get_admiral_option("subject_keys"), TRT01A, ...etc)`